Setup/SetupAssist/Checks/Domain/Test-ExchangeADSetupLevel.ps1

```diff
@@ -134,7 +134,7 @@ function Test-ExchangeADSetupLevel {
     }
 
     function GetExchangeADSetupLevel {
-        $rootDSE = [ADSI]("LDAP://RootDSE")
+        $rootDSE = [ADSI]("LDAP://$([System.DirectoryServices.ActiveDirectory.Domain]::GetComputerDomain().Name)/RootDSE")
```

Comment:
This is the same on line 148, could just set `$rootDSE` once then in this function.

Reply:
Good point. Not sure why that's in there twice.